client/client.go

```diff
@@ -182,3 +194,10 @@ func (x *PrmDial) SetTimeout(timeout time.Duration) {
 	x.timeoutDialSet = true
 	x.timeoutDial = timeout
 }
+
+// SetStreamTimeout sets the timeout for individual operations in streaming RPC.
+// MUST BE positive. If not called, 5s timeout will be used by default.
```

Comment:
Line 88 sets the timeout to `10s` by defatult

Reply:
Fixed